crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -380,8 +383,15 @@ impl<'t> AccountMinter<'t> {
                 error!(
                     "Couldn't create new source account, {:?}, try {}, retrying",
                     e, i
-                )
+                );
+                // random sleep to coordinate with other instances
+                std::thread::sleep(Duration::from_secs(rand::thread_rng().gen_range(0, 10)));
             } else {
+                new_source_account.set_sequence_number(
```

Comment:
This is unrelated?

Reply:
that's the base pr https://github.com/aptos-labs/aptos-core/pull/10837